### ISSUE TITLE
The config file of crowdhuman dataset

### ISSUE THREAD

**Issue by marvelcell** [hasanirtiza/PedesFormer#6]
Hello, thank you for your outstanding work, it helps me a lot!
But I am wondering which config file should I use to train the CrowdHuman dataset on swin-Transformer?
Thanks!

**Comment by hasanirtiza** [comment]
You can start with the same config as for cityperson for the crowdhuman. As far as I remember, we train it with the same configuration. If you do not get decent results, learning rate is the only thing I will suggest you to change. The issue is, I have left IIAI and unfortunately I do not have access to the crowdhuman config file anymore.

**Comment by marvelcell** [comment]
Thank you for your reply! I have successfully trained the crowdhuman dataset with the config of cityperson.
Sorry to bother you, but I still have a few questions:
1. It takes too long to train crowdhuman dataset on one GPU, is there any possible way to reduce the training time?
2. Is the config in the benchmarking is cityperson config you mentioned? I want to make sure that I used the right one.
3. Is the '.pth' file mentioned in /configs/_base_/default_runtime.py, line 15; is the .pth file download from here? :https://github.com/SwinTransformer/storage/releases/download/v1.0.2/cascade_mask_rcnn_swin_base_patch4_window7.pth
Thank you very much!!!

**Comment by hasanirtiza** [comment]
@marvelcell quick comment, how are the results after training.

1) I mean you can try the usual suspects, did you increase the batch size ? Or reducing the image resolution ? They usually speed up things. Else you can draw a small sample from the original crowdhuman data and see if and how much it effects.
2) Yes, the one mentioned in the table.
3) I think yes.

**Comment by marvelcell** [comment]
I haven't got the result yet, it still needs some time.
I will try your suggestions, thank you for answering my questions!

**Comment by marvelcell** [comment]
Hi! 
I'm sorry to bother you again.
I tried to use multiple GPUs to train the swinTRM and they are working well, but it still costs loads of time to gain the appropriate learning rate.
If possible, can you tell me the person I could contact to get the config of the crowdhuman dataset?
Thank you!

**Comment by hasanirtiza** [comment]
@marvelcell I am afraid, now no one would have this specific config or the logs related to the crowdhuman particular experiment.

**Comment by marvelcell** [comment]
All right, thank you for your help!

**Comment by marvelcell** [comment]
Hello, I am very sorry to bother you again. I am still researching the performance of Swin-transformer in detecting dense pedestrians and trying to improve its performance. I used CrowdHuman's entire training and validation set to train Swin-transformer+Cascade-Rcnn, but the mAP was 44.7, AP50 was 76.7, and AP75 was 45.1. The learning rate is 0.0001 at the beginning and will decay by 0.1 on the 27th and 35th epochs. May I ask if this is a good result in this model? Does it have any hope of further improvement? Thank you very much.
